### PR TITLE
Add Footer component

### LINKED ISSUE #168: Add footer
## Expected Behavior

There should be a footer with nav links
## Current Behavior

there is only a navbar
## Possible Solution

There is a `PR` @Kachulio1 created that does most of the hard work... maybe one can copy and paste his work and make some small css improvements?
See https://github.com/AgileVentures/WebsiteOne-FE/pull/137

Screenshot at 2019-03-26 00:15:19

### PR DESCRIPTION
**Pull request by arku**
Introduces a `Footer` component and adds it to the home page.

#### Issue addressed
Fixes #168

##### Desktop Footer

Desktop Footer

##### Mobile Footer
Mobile Footer

### CODE REVIEW

**Review by mattwr18 — CHANGES_REQUESTED** [review]
Looks good! As we talked about, I think the mobile view could probably be improved, but nothing broken, and it can be done as a separate ticket.

One thing I forgot to mention, could you please move the component to directly in the `components` directory? We have a couple components in their own directories, but they are outliers and really it would be nice to move them as well as we have almost every other component directly in the `components` directory.

The css files live in our `/assets` directory... Sorry for not clarifying in the ticket, I honestly forgot

**Comment by arku** [comment]
@mattwr18 No worries, I have updated the directory structure. PTAL :slightly_smiling_face:

**Review by mattwr18 — APPROVED** [review]
great, thanks!!

File: src/components/homepage/Homepage.js [hunk 1]
```diff
@@ -23,6 +24,7 @@ export class Homepage extends Component {
           <Grid.Row style={{ height: '320px' }}>
           </Grid.Row>
         </Grid>
+        <Footer />
```
Comment (mattwr18) [inline src/components/homepage/Homepage.js]: I think we should be rendering this on `App.js`?
I think we want the footer to appear on all pages

**Review by mattwr18 — CHANGES_REQUESTED** [review]

File: src/components/homepage/Homepage.js (hunk 1 above)
Comment (arku) [reply src/components/homepage/Homepage.js]: @mattwr18 Ah, you're right.

Let me fix it.

Comment (arku) [reply src/components/homepage/Homepage.js]: @mattwr18 Done!

Sorry for the inconvenience, can you PTAL?

Comment (mattwr18) [reply src/components/homepage/Homepage.js]: Awesome! thanks!! :smile_cat:

**Review by mattwr18 — APPROVED** [review]

**Review by joaopapereira — APPROVED** [review]
Looks good

**Pull request merged**